daemon/info.go

```diff
@@ -75,7 +75,7 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		DriverStatus:       daemon.layerStore.DriverStatus(),
 		Plugins:            daemon.showPluginsInfo(),
 		IPv4Forwarding:     !sysInfo.IPv4ForwardingDisabled,
-		BridgeNfIptables:   !sysInfo.BridgeNfCallIptablesDisabled,
+		BridgeNfIptables:   !sysInfo.BridgeNfCallIPtablesDisabled,
```

Comment:
@aaronlehmann  I agree we should treat `ip6tables` ad a word, but I use `Ip6tables` at https://github.com/docker/docker/pull/20692/files#diff-b2a8cebf4bface9ed8c0133cff941f83R90, it will make golint fail and it suggest to use `IP6tables`, so I think we can have this change.